entries/c/cohost.org.json

```diff
@@ -1,12 +1,7 @@
 {
   "cohost": {
     "domain": "cohost.org",
-    "contact": {
-      "email": "support@cohost.org",
-      "twitter": "cohost_org"
-    },
-    "categories": [
-      "social"
-    ]
+    "tfa": ["totp"],
+    "categories": ["social"]
```

Comment:
Please include one array value per line as this is the code style used for all the other files in the repo.